guide/results-viewer-react/package.json

```diff
@@ -63,14 +64,15 @@
     "@types/react-dom": "^18.0.10",
     "@types/react-transition-group": "^4.4.5",
     "@types/styled-components": "^5.1.26",
+    "@types/swagger2openapi": "^7.0.4",
     "@typescript-eslint/eslint-plugin": "^8.0.0",
     "@typescript-eslint/parser": "^8.0.0",
     "babel-loader": "^9.1.2",
     "eslint": "^9.8.0",
     "html-webpack-plugin": "^5.5.0",
+    "openapi-types": "^12.1.3",
     "storybook": "^8.2.5",
     "ts-loader": "^9.4.2",
-    "typescript": "^5.0.2",
```

Comment:
We need typescript as a dependency. Even though we don't directly use tsc, webpack is using it.